src/components/molecules/QualifyButton/index.js

```diff
@@ -0,0 +1,20 @@
+import { View, Text, Touchable } from 'react-native';
+import React from 'react';
+import { GeneralText, StarRating } from '../../atoms';
+import { StarPosition, TextCenter } from './styled';
+
+export const QualifyButton = () => {
+  return (
+    <StarPosition onPress={()=>console.log("Hola")/* abrir el pop up de del componente para escribir la opinion */}  >
```

Comment:
Don't use console.log() on GitHub, you can use [Alert.alert()](https://reactnative.dev/docs/alert) instead